### PR TITLE
Added a command "quick select bookmark from active group"

### PR DESCRIPTION
**Pull request by DragosStefanNecula**
This works as expected + I also added some code that accounts for an edge case (no bookmarks are available). Thank you for letting me make contribution to your extension! Hope you have a nice day. (Let me know if there is anything to fix in my code).

Related to issue #8

### CODE REVIEW

File: src/views/bookmark_x/controller.ts [hunk 1]
```diff
@@ -811,22 +811,30 @@ export class Controller {
...
+
+        if (cache == undefined) { return; }
+
+        let bookmarks = cache.values()
+            .filter(val => typeIsBookmarkLike(val.type))
+            .map(item => item.get_full_uri());
+
+        if (bookmarks.length === 0) {
+            vscode.window.showInformationMessage("no bookmarks available!"); return;
+        } else {
```
Comment (tommyme) [inline src/views/bookmark_x/controller.ts:829]: `Else` here is redundant and remove it can reduce the level of indentation.

**Review by tommyme — CHANGES_REQUESTED** [review]
there is a small problem, seems good generally.

**Comment by DragosStefanNecula** [comment]
Thank you!, I have changed the code as you requested

**Review by tommyme — COMMENTED** [review]
Thank you for your contribution! new beta version will be published in a week.

**Pull request merged**